test_czech_sort/test_data.py

```diff
@@ -41,6 +42,7 @@
     ['', ' ', '-', "'"],
     ['è', 'ê', 'ề'],
     ['a\n b', 'a \nb', 'a\nb', 'a b', 'ab'],
+    ['Ļ', 'Ł', 'M', 'Ơ', 'Ø', 'P'] # this line is new
```

Comment:
Tenhle komentář není potřeba – vo je nové můžeš zjistit pomocí Gitu (podle toho, jak jsi se změnou daleko: `git diff`, `git diff --cached`, `git diff HEAD`, ``git diff HEAD~`). Chceš to vysvětlit víc do hloubky?